src/exception/types.ts

```diff
@@ -137,3 +141,6 @@ export const isReadonlyArrayOf = <T>(
 ): input is ReadonlyArray<T> => {
   return input.every(validator);
 };
+
+// utility
```

Comment:
This gets used frequently, it deserves a better explanation for any TS newby that interacts with it.